modules/cas_cache/ocf_env.c

```diff
@@ -6,33 +6,60 @@
 #include "cas_cache.h"
 #include "utils/utils_rpool.h"
 
-/* *** ALLOCATOR *** */
-
+/** @addtogroup ALLOCATOR
+ * @{
+ */
+
+/**
+ * @def CAS_ALLOC_ALLOCATOR_LIMIT
+ * @brief default allocation limit for allocator in bytes
```

Comment:
I think this is not an allocation limit in bytes (allocator defines fixed-size allocations) but rather max number of allocated items

Reply:
thanks